packages/xchain-evm/package.json

```
@@ -36,15 +36,15 @@
     "directory": "release/package"
   },
   "dependencies": {
-    "@ledgerhq/hw-app-eth": "6.36.0",
+    "@ledgerhq/hw-app-eth": "6.45.4",
     "@xchainjs/xchain-client": "workspace:*",
     "@xchainjs/xchain-crypto": "workspace:*",
     "@xchainjs/xchain-evm-providers": "workspace:*",
     "@xchainjs/xchain-util": "workspace:*",
     "ethers": "5.7.2"
```

```suggestion
    "ethers": "^5.8.0"
```

While @ledgerhq/hw-app-eth is being updated, can this be updated in all the packages too?

---

will be updated to ethers v6